host/sizelimitTest.go

```diff
@@ -0,0 +1,191 @@
+// Copyright (c) 2016 Uber Technologies, Inc.
```

Comment:
Looks good to me. Could we have the updated header for all touched files?

Reply:
Of course, I think the linter forces us to do it.